### DISCUSSION TITLE
Is it possible to limit the memory usage DBADashService

### DISCUSSION THREAD

**Discussion by ArRiMartinez** [trimble-oss/dba-dash#478] [Q&A]
I am currently using DBADash with 19 connections.
The configuration for the moment is the default, with snapshot.
SQL versions range from 2005 to 2022.
The service runs in console mode with 10 threads.
The problem occurs in the memory consumption of the service, which starts at 200 Mb to 6 Gb, consuming 80% of the total.
**The question is the following: How to limit the memory consumption of the service to say 3 Gb.**

**Comment by DavidWiseman** [comment]
Question answered here #476
